modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/ChangeTrackingRenderView.js

```diff
@@ -790,33 +790,26 @@ export default function ChangeTrackingRenderView({
 	};
 
 	const navigate = (editURL, checkoutURL, confirmationMessage) => {
-		AUI().use('liferay-portlet-url', () => {
-			const editPortletURL = Liferay.PortletURL.createURL(editURL);
+		const editPortletURL = createPortletURL(editURL);
 
-			editPortletURL.setParameter(
-				'redirect',
-				window.location.pathname + window.location.search
-			);
+		editPortletURL.setParameter(
+			'redirect',
+			window.location.pathname + window.location.search
+		);
```

Comment:
I think that we can actually set these parameters inside of the initial `createPortletURL` invocation, but might not be something you want to do as part of this PR